polaris-intellisense/src/data/allTokens.ts

```diff
@@ -0,0 +1 @@
-surface-highlight-subdued-pressed","--p-text-highlight","--p-border-success","--p-border-success-subdued","--p-icon-success","--p-surface-success","--p-surface-success-subdued","--p-surface-success-subdued-hovered","--p-surface-success-subdued-pressed","--p-text-success","--p-decorative-one-icon","--p-decorative-one-surface","--p-decorative-one-text","--p-decorative-two-icon","--p-decorative-two-surface","--p-decorative-two-text","--p-decorative-three-icon","--p-decorative-three-surface","--p-decorative-three-text","--p-decorative-four-icon","--p-decorative-four-surface","--p-decorative-four-text","--p-decorative-five-icon","--p-decorative-five-surface","--p-decorative-five-text","--p-shadow-transparent","--p-shadow-faint","--p-shadow-base","--p-shadow-deep","--p-shadow-button","--p-shadow-top-bar","--p-shadow-card","--p-shadow-popover","--p-shadow-layer","--p-shadow-modal","--p-shadows-inset-button","--p-shadows-inset-button-pressed","--p-override-loading-z-index","--p-choice-size","--p-icon-size-small","--p-icon-size-medium","--p-choice-margin","--p-control-border-width","--p-banner-border-default","--p-banner-border-success","--p-banner-border-highlight","--p-banner-border-warning","--p-banner-border-critical","--p-thin-border-subdued","--p-text-field-spinner-offset","--p-text-field-focus-ring-offset","--p-button-group-item-spacing","--p-range-slider-thumb-size-base","--p-range-slider-thumb-size-active","--p-frame-offset","--p-duration-0","--p-duration-50","--p-duration-100","--p-duration-150","--p-duration-200","--p-duration-250","--p-duration-300","--p-duration-350","--p-duration-400","--p-duration-450","--p-duration-500","--p-ease","--p-ease-in","--p-ease-out","--p-ease-in-out","--p-linear","--p-border-radius-05","--p-border-radius-1","--p-border-radius-2","--p-border-radius-3","--p-border-radius-4","--p-border-radius-5","--p-border-radius-6","--p-border-radius-base","--p-border-radius-large","--p-border-radius-half","--p-border-width-1","--p-border-width-2","--p-border-width-3","--p-border-base","--p-border-dark","--p-border-transparent","--p-border-divider","--p-space-0","--p-space-025","--p-space-05","--p-space-1","--p-space-2","--p-space-3","--p-space-4","--p-space-5","--p-space-6","--p-space-8","--p-space-10","--p-space-12","--p-space-16","--p-space-20","--p-space-24","--p-space-28","--p-space-32","--p-font-family-sans","--p-font-family-mono","--p-font-size-1","--p-font-size-2","--p-font-size-3","--p-font-size-4","--p-font-size-5","--p-font-size-6","--p-font-size-7","--p-font-size-8","--p-font-size-9","--p-font-size-10","--p-font-size-11","--p-font-size-12","--p-font-size-13","--p-font-weight-regular","--p-font-weight-medium","--p-font-weight-semibold","--p-font-weight-bold","--p-line-height-1","--p-line-height-2","--p-line-height-3","--p-line-height-4","--p-line-height-5","--p-line-height-6","--p-line-height-7","--p-z-1","--p-z-2","--p-z-3","--p-z-4","--p-z-5","--p-z-6","--p-z-7","--p-z-8","--p-z-9","--p-z-10","--p-z-11","--p-z-12"];
```

Comment:
I would hope the reason for it being in this repository is so we don't need to have the list duplicated. Is there any reason we can't read the JSON files and convert them into custom properties?

Reply:
> Is there any reason we can't read the JSON files and convert them into custom properties?

We'll be able to do that when we reinstantiate `@shopify/polaris-tokens`, but the plugin won't have access to the tokens when deployed to the VSCode market place. e.g. `../../../src/tokens/token-groups/spacing.json` will not be accessible.

@yurm04 and I experimented with different TSC configurations and the bundle always resulted in relative paths to the token group files.

Reply:
One Google search put me on the right track and I think I figured out an alternative approach: https://github.com/Shopify/polaris-react/compare/add-polaris-intellisense..polaris-intellisense-experiment?diff=split

I tweaked the `tsconfig.json` to point to the root of `polaris-react` and reworked the custom property utils to directly import the token files. This allowed TSC to correctly build the module tree and output the appropriate files:
![image](https://user-images.githubusercontent.com/32409546/157358472-8aa90176-d182-477b-ad74-0ad9ea96247f.png)

Reply:
for the purposes of this experimentation phase I think keeping with the `generateCustomPropertyNames` script approach on prepublish suffices and is the easiest (because the work is already done 😄 ). The script will always run on prepublish of the package, so tokens should always be up to date whenever the project gets compiled. Since this is a temporary measure anyway until we can get our separate tokens package back in action, i think it's ok leaving things as they are for a bit